script.simkl/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.4"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
```

Comment:
xbmc.python version should be set at 2.24.0 to match jarvis. See: http://kodi.wiki/view/addon.xml